src/components/SuggestionsList/SuggestionsList.tsx

```diff
@@ -5,16 +5,18 @@ import { NestableBaseComponentProps } from '../../util/BaseComponent/props';
 import SuggestionsListItem, { SuggestionItem } from './SuggestionsListItem';
 import Spinner from '../Spinner';
 import Text, { TextSize, TextColor } from '../Text';
-import Callout, { DirectionalHint } from '../Callout';
 import Block, { GutterSize } from '../Block';
 import './SuggestionsList.css';
-import { IPoint } from 'office-ui-fabric-react/lib/utilities/positioning';
 
 export interface SuggestionsListProps extends NestableBaseComponentProps {
   /**
-   * The target that the Callout should try to position itself based on.
+   * The left offset position in pixels
    */
-  target: HTMLElement | IPoint;
+  left: number;
```

Comment:
This is not RTL-friendly.

Reply:
you're right :(. It's no less RTL friendly than our current stuff but I'll have a think about how we can solve it